app/controllers/tree_controller.rb

```diff
@@ -14,6 +14,11 @@ def automate_inline_methods
     render :body => json, :content_type => 'application/json'
   end
 
+  def ops_rbac
```

Comment:
You probably don't want this in this PR.

Reply:
ups soryy